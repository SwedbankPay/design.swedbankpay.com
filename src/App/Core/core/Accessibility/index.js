import React from "react";

const basename = process.env.basename;


const Intro = () => (
    <section id="intro">
        <p className="ingress font-weight-bold">At Swedbank Pay we value accessibility. Ourproducts are ment to be used by people no matter of their abilities and therefor westrive to be WCAG (Web Content Accessibility Guidelines) complient. We’re currentlyworking on our accessibility awareness and meeting the terms of the  accessibilitylaws in Norway and Sweden. </p>
        <p>Accessibility ties back to our values <span>simple</span> and <span>caring</span>.About 15 percent of the western world’s population is living with a disability of somekind. This translates to over 1 billion people, and includes visual, auditory,physical, speech, cognitive, and neurological disabilities. </p>

        <p>What is accessibility? Microsoft defines it like this: <span>“Accessibility: 1. Thequalities that make an experience open to all. 2. A professional discipline aimed atachieving No. 1”</span> So what do we meen when we talk about disabilities andimpairments? Well, a disability is a limitation when interacting in society inaccordance to how society is built. It can be permanent, temporary or situational –permanent could be someone with only one arm, temporary someone with a broken arm, andsituational a new parent (often carrying the baby in one arm). So, when we design forpermanent impairment and disabilities we automatically include people with bothtemporary and situational impairments as well.</p>

        <div className="img-container pb-4">
            <img src={`${basename}img/documentation/accessibility/impairments.svg`} alt="Illustration of impairments"/>
            <span><span className="font-weight-bold">Figure 1.</span> An illustration visualising permanent, temporary and situational impairment.</span>
        </div>

        <p className="mb-5">At Swedbank Pay we strive to give all customers with disabiliesand impairments (permanent, temporary and/or situaltional) a better chance of usingour services on the same terms as others. </p>
    </section>
);

const AccessibleService = () => (
    <section id="accessible-service">
        <h2>Why we want an accessible service</h2>
        <p className="mb-5">There are three main reasons as to why we think making our serviceaccessible makes sense. Firstly, it is the right thing to do. If that’s not enough toconvince us, there are also real business implications to excluding a large group ofyour user base due to not making your service accessible to them. Thirdly there areboth existing and future legal demands for accessibility, more on that later. We canprofit on making our service accessible. Designing for accessibility enables us tomeet our target group. Nobody needs to feel excluded from our services because of adisability.</p>
    </section>
);

const WcagStandard = () => (
    <section id="wcag-standard">
        <h2>WCAG standards and support material</h2>
        <p>WCAG has almost become a buzzword, thrown around by designers, developers and  stakeholders on regular basis. But what does it mean? WCAG is short for Web Content   Accessibility Guidelines and is a colletion of accessibility standards and support material developed and written by the W3C Web Accessibility Initiative (WAI). WCAG is    a work in progress with several different versions (see <a href="https://w3.org WAI">https://w3.org/WAI</a>).</p>
        <p>The guidelines ties back to four core terms: A website should be perceivable,  operable, understandable and robust. In these four areas there are sets of criterias  that has to be met for a certain term to be considered achieved (see Figure 2). Each  criteria also has one or more sub-criterias which are graded on a scale from A to AAA where A is least acceptable and AAA is considered “very good”.</p>

        <div className="img-container pb-4">
            <img src={`${basename}img/documentation/accessibility/wcag.svg`} className="mb-1" alt="WCAG 2.1 at a glance" />
            <span><span className="font-weight-bold">Figure 2.</span> This is the WCAG 2.1 at a glance from <a href="https://www.w3.org/WAI/standards-guidelines/wcag/glance/ ">www.w3.org/WAI/standards-guidelines/wcag/glance/ </a></span>
        </div>

        <h3 className="mt-0">More than a checklist</h3>
        <p>Important to remember is that WCAG and accessibility is more than a checklist! It  sure can start with a checklist (such as this one) but it is a practice and mindset   that must be present amongst all product employees. To meet the whole target group these terms are nessessary to meet. </p>
        <a href="https://www.w3.org/WAI/standards-guidelines/wcag/" className="icon-link  d-block"><i className="material-icons mr-2" aria-hidden="true">open_in_new</i><span className="mr-2">What is WCAG?</span></a>

        <a href="https://www.w3.org/WAI/standards-guidelines/wcag/glance/" className="icon-link mb-5"><i className="material-icons mr-2" aria-hidden="true">open_in_new</i><span className="mr-2">W3C – WCAG 2.1    Recommendations </span></a>
    </section>
);

const Inclusions = () => (
    <section id="inclusions">
        <h2>Creating inclusion</h2>
        <p>Today we’re working with awareness. All designers and developers at Swedbank Pay should be aware of accessibility, what it is, how it’s implemented and, what procations we have to take to acctually follow the law in the countries we’re in. As mentioned, accessibility may start with design but the main part is making our service compatible with accessibility helping tools, and that part happens in the technical implementation. </p>
        <p className="mb-2">Accessibility and WCAG can seem overwhelming and just as with everything else we need to take it step by step. This is a list from a Funka webinar held 2020-11-06:</p>
        <ul className="pl-3 mb-5">
            <li>Make sure that we know the current status of our products. Are we accessible enough?</li>
            <li>Planning and prioritising what we need to do to get where we need to be.</li>
            <li>Set up requirements and take control. Make sure that all employees knows what being accessible means. </li>
            <li>Create internal processes to sustainably work with accessibility. What’s our plan?</li>
            <li>We need to build internal compentence, not soly rely on external experts such as Funka.</li>
        </ul>
    </section>
);

const DiveDeeper = () => (
    <section id="dive-deeper" className="mb-5">
        <h2>Dive deeper into accessibility</h2>
        <p>Accessibility can be an ocean and we know it can be hard to navigate it – that’s at least what we in the design team think. We can always improve the user experience, and thereby our products, by making them accessible for more people. Here are some links where you can read more about accessibility and practices to achieve inclusivness for as many users as possible. </p>
        <a href="https://material.io/design/usability/accessibility.html#understanding-accessibility" className="icon-link d-block"><i className="material-icons mr-2" aria-hidden="true">open_in_new</i><span className="mr-2">Material.io – Understandinig Accessibility </span></a>
        <a href="https://www.w3.org/WAI/standards-guidelines/wcag/glance/" className="icon-link d-block"><i className="material-icons mr-2" aria-hidden="true">open_in_new</i><span className="mr-2">W3C – WCAG 2.1 Recommendations</span></a>

        <h3 className="my-3">Website accessibility and the law</h3>
        <a href="https://www.funka.com/design-for-alla/lagar-och-regler/" className="icon-link d-block"><i className="material-icons mr-2" aria-hidden="true">open_in_new</i><span className="mr-2">Funka.com – Lagar och regler</span></a>
        <a href="https://www.funka.com/en/design-for-all/accessibility/web-accessibility-directive-norway-is-facing-a-choice/" className="icon-link d-block"><i className="material-icons mr-2" aria-hidden="true">open_in_new</i><span className="mr-2"> Funka.com – Web accessibility directive: Norway is facing a choice</span></a>
        <a href="https://medium.com/confrere/its-illegal-to-have-an-inaccessible-website-in-norway-and-that-s-good-news-for-all-of-us-b59a9e929d54" className="icon-link d-block"><i className="material-icons mr-2" aria-hidden="true">open_in_new</i><span className="mr-2">Medium.com – It’s illegal to have an inaccessible website in Norway and that’s good for all of us</span></a>

        <h3 className="my-3">Others about accessibility</h3>
        <a href="https://www.swedbank.com/sv/hallbarhet/ansvarsfulla-affarer/betala.html" className="icon-link d-block"><i className="material-icons mr-2" aria-hidden="true">open_in_new</i><span className="mr-2">Swedbank – Ansvarsfulla affärer </span></a>
        <a href="https://polaris.shopify.com/foundations/accessibility" className="icon-link d-block"><i className="material-icons mr-2" aria-hidden="true">open_in_new</i><span className="mr-2">Shopify – Accessibility</span></a>
        <a href="https://material.io/design/usability/accessibility.html" className="icon-link d-block"><i className="material-icons mr-2" aria-hidden="true">open_in_new</i><span className="mr-2">Google Material – Accessibility</span></a>
        <a href="http://www.lightningdesignsystem.com/accessibility/overview/" className="icon-link d-block"><i className="material-icons mr-2" aria-hidden="true">open_in_new</i><span className="mr-2">Salesforce Lightning – Accessibility</span></a>
        <a href="https://spotify.design/article/spotify-inclusive-design-global-accessibility-awareness-day-round-up" className="icon-link d-block"><i className="material-icons mr-2" aria-hidden="true">open_in_new</i><span className="mr-2">Spotify – Inclusive design global accessibility awareness day round up </span></a>
    </section>
);

const Accessibility = () => (

    <div className="doc-body accessibility mx-auto p-0">
        <Intro />
        <AccessibleService />
        <WcagStandard />
        <Inclusions />
        <DiveDeeper />
        <span className="font-italic">This text was last updated 2020-11-24</span>
    </div>
);

export default Accessibility;

export { Intro, AccessibleService, WcagStandard, Inclusions, DiveDeeper };
