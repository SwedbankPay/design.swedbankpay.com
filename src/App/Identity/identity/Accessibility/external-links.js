export const accessibilityAndLaw = [
    {
        url: "https://www.funka.com/design-for-alla/lagar-och-regler/",
        text: "Funka.com – Lagar och regler"
    },
    {
        url: "https://www.funka.com/en/design-for-all/accessibility/web-accessibility-directive-norway-is-facing-a-choice/",
        text: "Funka.com – Web accessibility directive: Norway is facing a choice"
    },
    {
        url: "https://medium.com/confrere/its-illegal-to-have-an-inaccessible-website-in-norway-and-that-s-good-news-for-all-of-us-b59a9e929d54",
        text: "Medium.com – It’s illegal to have an inaccessible website in Norway and that’s good for all of us"
    }
];

export const othersAboutAccessibility = [
    {
        url: "https://www.swedbank.com/sv/hallbarhet/ansvarsfulla-affarer/betala.html",
        text: "Swedbank – Ansvarsfulla affärer"
    },
    {
        url: "https://polaris.shopify.com/foundations/accessibility",
        text: "Shopify – Accessibility"
    },
    {
        url: "https://material.io/design/usability/accessibility.html",
        text: "Google Material – Accessibility"
    },
    {
        url: "http://www.lightningdesignsystem.com/accessibility/overview/",
        text: "Salesforce Lightning – Accessibility"
    },
    {
        url: "https://spotify.design/article/spotify-inclusive-design-global-accessibility-awareness-day-round-up",
        text: "Spotify – Inclusive design global accessibility awareness day round up"
    }
];
