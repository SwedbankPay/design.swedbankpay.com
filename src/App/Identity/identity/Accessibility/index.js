import React from "react";
import * as externalLinks from "./constants";

import { DocContainer } from "@docutils";

const basename = process.env.basename;

const Intro = () => (
    <section>
        <p className="lead">At Swedbank Pay we value accessibility. Our products are meant to be used by people no matter of their abilities and therefor we strive to be WCAG (Web Content Accessibility Guidelines) compliant. We’re currently working on our accessibility awareness and meeting the terms of the  accessibility laws in Norway and Sweden. </p>
        <h2>What is accessibility</h2>
        <p>Accessibility ties back to our values <span>simple</span> and <span>caring</span>.About 15 percent of the western world’s population is living with a disability of some kind. This translates to over 1 billion people, and includes visual, auditory, physical, speech, cognitive, and neurological disabilities. </p>

        <p>Microsoft defines it like this: <span>“Accessibility: 1. The qualities that make an experience open to all. 2. A professional discipline aimed at achieving No. 1”</span> So what do we mean when we talk about disabilities and impairments? Well, a disability is a limitation when interacting in society in accordance to how society is built. It can be permanent, temporary or situational –permanent could be someone with only one arm, temporary someone with a broken arm, and situational a new parent (often carrying the baby in one arm). So, when we design for permanent impairment and disabilities we automatically include people with both temporary and situational impairments as well.</p>

        <div className="d-flex flex-column align-items-center mb-4">
            <img src={`${basename}img/documentation/accessibility/impairments.png`} className="w-100" alt="Illustration of impairments"/>
            <span className="caption-text"><span className="font-weight-bold">Figure 1.</span> An illustration visualizing permanent, beneath temporary and situational impairment.</span>
        </div>

        <p>At Swedbank Pay we strive to give all customers with disabilities and impairments (permanent, temporary and/or situational) a better chance of using our services on the same terms as others. </p>
    </section>
);

const AccessibleService = () => (
    <section>
        <h2 id="accessible-service">Why we want an accessible service</h2>
        <p>There are three main reasons as to why we think making our service accessible makes sense. Firstly, it is the right thing to do. If that’s not enough to convince us, there are also real business implications to excluding a large group of your user base due to not making your service accessible to them. Thirdly there are both existing and future legal demands for accessibility, more on that later. We can profit on making our service accessible. Designing for accessibility enables us to meet our target group. Nobody needs to feel excluded from our services because of a disability.</p>
    </section>
);

const WcagStandard = () => (
    <section>
        <h2 id="wcag-standard">WCAG standards and support material</h2>
        <p>WCAG has almost become a buzzword, thrown around by designers, developers and  stakeholders on regular basis. But what does it mean? WCAG is short for Web Content   Accessibility Guidelines and is a collection of accessibility standards and support material developed and written by the W3C Web Accessibility Initiative (WAI). WCAG is    a work in progress with several different versions (see <a href="https://www.w3.org/WAI/" target="_blank" rel="noopener noreferrer">https://w3.org/WAI</a>).</p>
        <p>The guidelines ties back to four core terms: A website should be perceivable,  operable, understandable and robust. In these four areas there are sets of criteria  that has to be met for a certain term to be considered achieved (see Figure 2). Each  criteria also has one or more sub-criteria which are graded on a scale from A to AAA where A is least acceptable and AAA is considered “very good”.</p>

        <div className="d-flex flex-column align-items-center">
            <img src={`${basename}img/documentation/accessibility/wcag.png`} className="mb-1 w-100" alt="WCAG 2.1 at a glance" />
            <span className="caption-text"><span className="font-weight-bold">Figure 1</span> This is the WCAG 2.1 at a glance from <a href="https://www.w3.org/WAI/standards-guidelines/wcag/glance/" target="_blank" rel="noopener noreferrer">www.w3.org/WAI/standards-guidelines/wcag/glance/ </a></span>
        </div>

        <h3>More than a checklist</h3>
        <p>Important to remember is that WCAG and accessibility is more than a checklist! It  sure can start with a checklist (such as this one) but it is a practice and mindset   that must be present amongst all product employees. To meet the whole target group these terms are necessary to meet. </p>
        <a href="https://www.w3.org/WAI/standards-guidelines/wcag/" target="_blank" rel="noopener noreferrer" className="icon-link d-flex">
            <i className="material-icons mr-2" aria-hidden="true">open_in_new</i>
            <span className="mr-2">What is WCAG?</span>
        </a>

        <a href="https://www.w3.org/WAI/standards-guidelines/wcag/glance/" target="_blank" rel="noopener noreferrer" className="icon-link">
            <i className="material-icons mr-2" aria-hidden="true">open_in_new</i>
            <span className="mr-2">W3C – WCAG 2.1    Recommendations </span>
        </a>
    </section>
);

const Inclusions = () => (
    <section>
        <h2 id="inclusions">Creating inclusion</h2>
        <p>Today we’re working with awareness. All designers and developers at Swedbank Pay should be aware of accessibility, what it is, how it’s implemented and, what precautions we have to take to actually follow the law in the countries we’re in. As mentioned, accessibility may start with design but the main part is making our service compatible with accessibility helping tools, and that part happens in the technical implementation. </p>
        <p>Accessibility and WCAG can seem overwhelming and just as with everything else we need to take it step by step. This is a list from a Funka webinar held 2020-11-06:</p>
        <ul className="list list-bullet">
            <li>Make sure that we know the current status of our products. Are we accessible enough?</li>
            <li>Planning and prioritizing what we need to do to get where we need to be.</li>
            <li>Set up requirements and take control. Make sure that all employees knows what being accessible means. </li>
            <li>Create internal processes to sustainably work with accessibility. What’s our plan?</li>
            <li>We need to build internal competence, not solely rely on external experts such as Funka.</li>
        </ul>
    </section>
);

const DiveDeeper = () => (
    <section>
        <h2 id="dive-deeper">Dive deeper into accessibility</h2>
        <p>Accessibility can be an ocean and we know it can be hard to navigate it – that’s at least what we in the design team think. We can always improve the user experience, and thereby our products, by making them accessible for more people. Here are some links where you can read more about accessibility and practices to achieve inclusiveness for as many users as possible. </p>
        <a href="https://material.io/design/usability/accessibility.html#understanding-accessibility" target="_blank" rel="noopener noreferrer" className="icon-link d-flex">
            <i className="material-icons mr-2" aria-hidden="true">open_in_new</i>
            <span className="mr-2">Material.io – Understanding Accessibility </span>
        </a>
        <a href="https://www.w3.org/WAI/standards-guidelines/wcag/glance/" target="_blank" rel="noopener noreferrer" className="icon-link d-flex">
            <i className="material-icons mr-2" aria-hidden="true">open_in_new</i>
            <span className="mr-2">W3C – WCAG 2.1 Recommendations</span>
        </a>

        <h3>Website accessibility and the law</h3>
        {externalLinks.accessibilityAndLaw.map(links => (
            <a key={links.url} href={links.url} target="_blank" rel="noopener noreferrer" className="icon-link d-flex">
                <i className="material-icons mr-2" aria-hidden="true">open_in_new</i>
                <span className="mr-2">{links.text}</span>
            </a>
        ))}

        <h3>Others about accessibility</h3>
        {externalLinks.othersAboutAccessibility.map(links => (
            <a key={links.url} href={links.url} target="_blank" rel="noopener noreferrer" className="icon-link d-flex">
                <i className="material-icons mr-2" aria-hidden="true">open_in_new</i>
                <span className="mr-2">{links.text}</span>
            </a>
        ))}
    </section>
);

const Accessibility = () => (
    <DocContainer>
        <Intro />
        <AccessibleService />
        <WcagStandard />
        <Inclusions />
        <DiveDeeper />
        <span className="font-italic">This text was last updated 2020-11-24</span>
    </DocContainer>
);

export default Accessibility;

export { Intro, AccessibleService, WcagStandard, Inclusions, DiveDeeper };
