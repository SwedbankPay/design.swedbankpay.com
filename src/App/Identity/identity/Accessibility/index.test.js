import React from "react";
import renderer from "react-test-renderer";

import Accessibility, {
	Intro,
	AccessibleService,
	WcagStandard,
	Inclusions,
	DiveDeeper,
} from "./index";

describe("Core: Accessibility", () => {
	it("is defined", () => {
		expect(Accessibility).toBeDefined();
	});

	describe("Intro", () => {
		it("is defined", () => {
			expect(Intro).toBeDefined();
		});
	});

	describe("AccessibleService", () => {
		it("is defined", () => {
			expect(AccessibleService).toBeDefined();
		});

		it("renders", () => {
			const wrapper = renderer.create(<AccessibleService />);

			expect(wrapper.toJSON()).toMatchSnapshot();
		});
	});

	describe("WcagStandard", () => {
		it("is defined", () => {
			expect(WcagStandard).toBeDefined();
		});

		it("renders", () => {
			const wrapper = renderer.create(<WcagStandard />);

			expect(wrapper.toJSON()).toMatchSnapshot();
		});
	});

	describe("Inclusions", () => {
		it("is defined", () => {
			expect(Inclusions).toBeDefined();
		});

		it("renders", () => {
			const wrapper = renderer.create(<Inclusions />);

			expect(wrapper.toJSON()).toMatchSnapshot();
		});
	});

	describe("DiveDeeper", () => {
		it("is defined", () => {
			expect(DiveDeeper).toBeDefined();
		});

		it("renders", () => {
			const wrapper = renderer.create(<DiveDeeper />);

			expect(wrapper.toJSON()).toMatchSnapshot();
		});
	});
});
