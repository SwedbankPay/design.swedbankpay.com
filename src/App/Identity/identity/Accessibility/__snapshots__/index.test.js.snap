// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Core: Accessibility AccessibleService renders 1`] = `
<section>
  <h2
    id="accessible-service"
  >
    Why we want an accessible service
  </h2>
  <p>
    There are three main reasons as to why we think making our service accessible makes sense. Firstly, it is the right thing to do. If that’s not enough to convince us, there are also real business implications to excluding a large group of your user base due to not making your service accessible to them. Thirdly there are both existing and future legal demands for accessibility, more on that later. We can profit on making our service accessible. Designing for accessibility enables us to meet our target group. Nobody needs to feel excluded from our services because of a disability.
  </p>
</section>
`;

exports[`Core: Accessibility DiveDeeper renders 1`] = `
<section>
  <h2
    id="dive-deeper"
  >
    Dive deeper into accessibility
  </h2>
  <p>
    Accessibility can be an ocean and we know it can be hard to navigate it – that’s at least what we in the design team think. We can always improve the user experience, and thereby our products, by making them accessible for more people. Here are some links where you can read more about accessibility and practices to achieve inclusiveness for as many users as possible. 
  </p>
  <a
    className="icon-link d-flex"
    href="https://material.io/design/usability/accessibility.html#understanding-accessibility"
    rel="noopener noreferrer"
    target="_blank"
  >
    <i
      aria-hidden="true"
      className="material-icons mr-2"
    >
      open_in_new
    </i>
    <span
      className="mr-2"
    >
      Material.io – Understanding Accessibility 
    </span>
  </a>
  <a
    className="icon-link d-flex"
    href="https://www.w3.org/WAI/standards-guidelines/wcag/glance/"
    rel="noopener noreferrer"
    target="_blank"
  >
    <i
      aria-hidden="true"
      className="material-icons mr-2"
    >
      open_in_new
    </i>
    <span
      className="mr-2"
    >
      W3C – WCAG 2.1 Recommendations
    </span>
  </a>
  <h3>
    Website accessibility and the law
  </h3>
  <a
    className="icon-link d-flex"
    href="https://www.funka.com/design-for-alla/lagar-och-regler/"
    key="https://www.funka.com/design-for-alla/lagar-och-regler/"
    rel="noopener noreferrer"
    target="_blank"
  >
    <i
      aria-hidden="true"
      className="material-icons mr-2"
    >
      open_in_new
    </i>
    <span
      className="mr-2"
    >
      Funka.com – Lagar och regler
    </span>
  </a>
  <a
    className="icon-link d-flex"
    href="https://www.funka.com/en/design-for-all/accessibility/web-accessibility-directive-norway-is-facing-a-choice/"
    key="https://www.funka.com/en/design-for-all/accessibility/web-accessibility-directive-norway-is-facing-a-choice/"
    rel="noopener noreferrer"
    target="_blank"
  >
    <i
      aria-hidden="true"
      className="material-icons mr-2"
    >
      open_in_new
    </i>
    <span
      className="mr-2"
    >
      Funka.com – Web accessibility directive: Norway is facing a choice
    </span>
  </a>
  <a
    className="icon-link d-flex"
    href="https://medium.com/confrere/its-illegal-to-have-an-inaccessible-website-in-norway-and-that-s-good-news-for-all-of-us-b59a9e929d54"
    key="https://medium.com/confrere/its-illegal-to-have-an-inaccessible-website-in-norway-and-that-s-good-news-for-all-of-us-b59a9e929d54"
    rel="noopener noreferrer"
    target="_blank"
  >
    <i
      aria-hidden="true"
      className="material-icons mr-2"
    >
      open_in_new
    </i>
    <span
      className="mr-2"
    >
      Medium.com – It’s illegal to have an inaccessible website in Norway and that’s good for all of us
    </span>
  </a>
  <h3>
    Others about accessibility
  </h3>
  <a
    className="icon-link d-flex"
    href="https://www.swedbank.com/sv/hallbarhet/ansvarsfulla-affarer/betala.html"
    key="https://www.swedbank.com/sv/hallbarhet/ansvarsfulla-affarer/betala.html"
    rel="noopener noreferrer"
    target="_blank"
  >
    <i
      aria-hidden="true"
      className="material-icons mr-2"
    >
      open_in_new
    </i>
    <span
      className="mr-2"
    >
      Swedbank – Ansvarsfulla affärer
    </span>
  </a>
  <a
    className="icon-link d-flex"
    href="https://polaris.shopify.com/foundations/accessibility"
    key="https://polaris.shopify.com/foundations/accessibility"
    rel="noopener noreferrer"
    target="_blank"
  >
    <i
      aria-hidden="true"
      className="material-icons mr-2"
    >
      open_in_new
    </i>
    <span
      className="mr-2"
    >
      Shopify – Accessibility
    </span>
  </a>
  <a
    className="icon-link d-flex"
    href="https://material.io/design/usability/accessibility.html"
    key="https://material.io/design/usability/accessibility.html"
    rel="noopener noreferrer"
    target="_blank"
  >
    <i
      aria-hidden="true"
      className="material-icons mr-2"
    >
      open_in_new
    </i>
    <span
      className="mr-2"
    >
      Google Material – Accessibility
    </span>
  </a>
  <a
    className="icon-link d-flex"
    href="http://www.lightningdesignsystem.com/accessibility/overview/"
    key="http://www.lightningdesignsystem.com/accessibility/overview/"
    rel="noopener noreferrer"
    target="_blank"
  >
    <i
      aria-hidden="true"
      className="material-icons mr-2"
    >
      open_in_new
    </i>
    <span
      className="mr-2"
    >
      Salesforce Lightning – Accessibility
    </span>
  </a>
  <a
    className="icon-link d-flex"
    href="https://spotify.design/article/spotify-inclusive-design-global-accessibility-awareness-day-round-up"
    key="https://spotify.design/article/spotify-inclusive-design-global-accessibility-awareness-day-round-up"
    rel="noopener noreferrer"
    target="_blank"
  >
    <i
      aria-hidden="true"
      className="material-icons mr-2"
    >
      open_in_new
    </i>
    <span
      className="mr-2"
    >
      Spotify – Inclusive design global accessibility awareness day round up
    </span>
  </a>
</section>
`;

exports[`Core: Accessibility Inclusions renders 1`] = `
<section>
  <h2
    id="inclusions"
  >
    Creating inclusion
  </h2>
  <p>
    Today we’re working with awareness. All designers and developers at Swedbank Pay should be aware of accessibility, what it is, how it’s implemented and, what precautions we have to take to actually follow the law in the countries we’re in. As mentioned, accessibility may start with design but the main part is making our service compatible with accessibility helping tools, and that part happens in the technical implementation. 
  </p>
  <p>
    Accessibility and WCAG can seem overwhelming and just as with everything else we need to take it step by step. This is a list from a Funka webinar held 2020-11-06:
  </p>
  <ul
    className="list list-bullet"
  >
    <li>
      Make sure that we know the current status of our products. Are we accessible enough?
    </li>
    <li>
      Planning and prioritizing what we need to do to get where we need to be.
    </li>
    <li>
      Set up requirements and take control. Make sure that all employees knows what being accessible means. 
    </li>
    <li>
      Create internal processes to sustainably work with accessibility. What’s our plan?
    </li>
    <li>
      We need to build internal competence, not solely rely on external experts such as Funka.
    </li>
  </ul>
</section>
`;

exports[`Core: Accessibility Intro renders 1`] = `
<section>
  <p
    className="lead"
  >
    At Swedbank Pay we value accessibility. Our products are meant to be used by people no matter of their abilities and therefor we strive to be WCAG (Web Content Accessibility Guidelines) compliant. We’re currently working on our accessibility awareness and meeting the terms of the  accessibility laws in Norway and Sweden. 
  </p>
  <h2>
    What is accessibility
  </h2>
  <p>
    Accessibility ties back to our values 
    <span>
      simple
    </span>
     and 
    <span>
      caring
    </span>
    .About 15 percent of the western world’s population is living with a disability of some kind. This translates to over 1 billion people, and includes visual, auditory, physical, speech, cognitive, and neurological disabilities. 
  </p>
  <p>
    Microsoft defines it like this: 
    <span>
      “Accessibility: 1. The qualities that make an experience open to all. 2. A professional discipline aimed at achieving No. 1”
    </span>
     So what do we mean when we talk about disabilities and impairments? Well, a disability is a limitation when interacting in society in accordance to how society is built. It can be permanent, temporary or situational –permanent could be someone with only one arm, temporary someone with a broken arm, and situational a new parent (often carrying the baby in one arm). So, when we design for permanent impairment and disabilities we automatically include people with both temporary and situational impairments as well.
  </p>
  <div
    className="d-flex flex-column align-items-center mb-4"
  >
    <img
      alt="Illustration of impairments"
      className="w-100"
      src="/TEST/img/documentation/accessibility/impairments.png"
    />
    <span
      className="caption-text"
    >
      <span
        className="font-weight-bold"
      >
        Figure 1.
      </span>
       An illustration visualizing permanent, beneath temporary and situational impairment.
    </span>
  </div>
  <p>
    At Swedbank Pay we strive to give all customers with disabilities and impairments (permanent, temporary and/or situational) a better chance of using our services on the same terms as others. 
  </p>
</section>
`;

exports[`Core: Accessibility WcagStandard renders 1`] = `
<section>
  <h2
    id="wcag-standard"
  >
    WCAG standards and support material
  </h2>
  <p>
    WCAG has almost become a buzzword, thrown around by designers, developers and  stakeholders on regular basis. But what does it mean? WCAG is short for Web Content   Accessibility Guidelines and is a collection of accessibility standards and support material developed and written by the W3C Web Accessibility Initiative (WAI). WCAG is    a work in progress with several different versions (see 
    <a
      href="https://www.w3.org/WAI/"
      rel="noopener noreferrer"
      target="_blank"
    >
      https://w3.org/WAI
    </a>
    ).
  </p>
  <p>
    The guidelines ties back to four core terms: A website should be perceivable,  operable, understandable and robust. In these four areas there are sets of criteria  that has to be met for a certain term to be considered achieved (see Figure 2). Each  criteria also has one or more sub-criteria which are graded on a scale from A to AAA where A is least acceptable and AAA is considered “very good”.
  </p>
  <div
    className="d-flex flex-column align-items-center"
  >
    <img
      alt="WCAG 2.1 at a glance"
      className="mb-1 w-100"
      src="/TEST/img/documentation/accessibility/wcag.png"
    />
    <span
      className="caption-text"
    >
      <span
        className="font-weight-bold"
      >
        Figure 1
      </span>
       This is the WCAG 2.1 at a glance from 
      <a
        href="https://www.w3.org/WAI/standards-guidelines/wcag/glance/"
        rel="noopener noreferrer"
        target="_blank"
      >
        www.w3.org/WAI/standards-guidelines/wcag/glance/ 
      </a>
    </span>
  </div>
  <h3>
    More than a checklist
  </h3>
  <p>
    Important to remember is that WCAG and accessibility is more than a checklist! It  sure can start with a checklist (such as this one) but it is a practice and mindset   that must be present amongst all product employees. To meet the whole target group these terms are necessary to meet. 
  </p>
  <a
    className="icon-link d-flex"
    href="https://www.w3.org/WAI/standards-guidelines/wcag/"
    rel="noopener noreferrer"
    target="_blank"
  >
    <i
      aria-hidden="true"
      className="material-icons mr-2"
    >
      open_in_new
    </i>
    <span
      className="mr-2"
    >
      What is WCAG?
    </span>
  </a>
  <a
    className="icon-link"
    href="https://www.w3.org/WAI/standards-guidelines/wcag/glance/"
    rel="noopener noreferrer"
    target="_blank"
  >
    <i
      aria-hidden="true"
      className="material-icons mr-2"
    >
      open_in_new
    </i>
    <span
      className="mr-2"
    >
      W3C – WCAG 2.1    Recommendations 
    </span>
  </a>
</section>
`;

exports[`Core: Accessibility renders 1`] = `
<DocContainer>
  <Intro />
  <AccessibleService />
  <WcagStandard />
  <Inclusions />
  <DiveDeeper />
  <span
    className="font-italic"
  >
    This text was last updated 2020-11-24
  </span>
</DocContainer>
`;
